src/Typography/typography.module.scss

```diff
@@ -17,7 +17,6 @@
 .h1 {
   font-size: 44px;
   line-height: 48px;
-  font-weight: 900;
```

Comment:
Are we sure this is correct? On Mattress Matcher and Delivery Tracker, the h1 tag is 900 font weight

Reply:
Yes this is the font for our design guide for h1s. Sometimes we might need adjusted styles not from the style guide (like yeah, our only two apps so far....) So that would be an argument to add more styles to the style guide, but this is what it should be defined as for our Typography, confirmed this with Isaac

Reply:
lol ok